server.js

```diff
@@ -33,6 +33,24 @@ function Location (locationData, cityDescrip){
   this.longitude = locationData[0].lon;
 }
 
+app.get('/restaurants', restaurantHandler);
+
+function restaurantHandler(req,res){
+  const restaurantJSON = require('.data/restaturants.json');
+
+  const output = [];
+  for (let i = 0; i < restaurantJSON.nearby_restaurants.length; i++){
+    output.push(new Restaurant(restaurantJSON.nearby_restaurants[i].restaurant))
```

Comment:
I like how you iterated over the json's properties all at once in one line

Reply:
Did you include the Restaurant Code just for the demo, or is this also part of the lab?

Reply:
Can create weather without needed to use "req" as an argument.